enderio-base/src/main/java/com/enderio/base/data/recipe/standard/MaterialRecipes.java

```diff
@@ -29,5 +41,231 @@ protected void buildCraftingRecipes(@Nonnull Consumer<FinishedRecipe> recipeCons
+        blockToIngots(recipeConsumer, EIOItems.VIBRANT_ALLOY_INGOT.get(),EIOBlocks.VIBRANT_ALLOY_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.VIBRANT_ALLOY_NUGGET.get(), EIOItems.VIBRANT_ALLOY_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.REDSTONE_ALLOY_INGOT.get(),EIOBlocks.REDSTONE_ALLOY_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.REDSTONE_ALLOY_NUGGET.get(), EIOItems.REDSTONE_ALLOY_INGOT.get()); 
+        blockToIngots(recipeConsumer, EIOItems.CONDUCTIVE_IRON_INGOT.get(),EIOBlocks.CONDUCTIVE_IRON_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.CONDUCTIVE_IRON_NUGGET.get(), EIOItems.CONDUCTIVE_IRON_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.PULSATING_IRON_INGOT.get(),EIOBlocks.PULSATING_IRON_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.PULSATING_IRON_NUGGET.get(), EIOItems.PULSATING_IRON_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.DARK_STEEL_INGOT.get(),EIOBlocks.DARK_STEEL_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.DARK_STEEL_NUGGET.get(), EIOItems.DARK_STEEL_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.SOULARIUM_INGOT.get(),EIOBlocks.SOULARIUM_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.SOULARIUM_NUGGET.get(), EIOItems.SOULARIUM_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.END_STEEL_INGOT.get(),EIOBlocks.END_STEEL_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.END_STEEL_NUGGET.get(), EIOItems.END_STEEL_INGOT.get());
+        blockToIngots(recipeConsumer, EIOItems.CONSTRUCTION_ALLOY_INGOT.get(),EIOBlocks.CONSTRUCTION_ALLOY_BLOCK.get());
+        ingotToNuggets(recipeConsumer, EIOItems.CONSTRUCTION_ALLOY_NUGGET.get(), EIOItems.CONSTRUCTION_ALLOY_INGOT.get());
+        
+        ShapedRecipeBuilder.shaped(EIOItems.CONDUIT_BINDER_COMPOSITE.get())
+            .pattern("GCG")
+            .pattern("SGS")
+            .pattern("GCG")
+            .define('G', Tags.Items.GRAVEL)
+            .define('S', Tags.Items.SAND)
+            .define('C', Items.CLAY_BALL)
+            .unlockedBy("has_ingredient_gravel", InventoryChangeTrigger.TriggerInstance.hasItems(Items.GRAVEL))
+            .unlockedBy("has_ingredient_sand", InventoryChangeTrigger.TriggerInstance.hasItems(Items.SAND))
+            .unlockedBy("has_ingredient_clay", InventoryChangeTrigger.TriggerInstance.hasItems(Items.CLAY_BALL))
+            .save(recipeConsumer);
+        
+        ShapedRecipeBuilder.shaped(EIOItems.PULSATING_CRYSTAL.get())
+            .pattern("PPP")
+            .pattern("PDP")
+            .pattern("PPP")
+            .define('P', EIOItems.PULSATING_IRON_NUGGET.get())
+            .define('D', Tags.Items.GEMS_DIAMOND)
+            .unlockedBy("has_ingredien", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.PULSATING_IRON_NUGGET.get()))
```

Comment:
Just noticed a typo across the file: needs changing from `has_ingredien` -> `has_ingredient`